reconcile/status.py

```diff
@@ -20,3 +20,7 @@ class RunningState(_RunningState):
     the running state. Attributes will be populated
     by the callers.
     """
+    def __getattr__(self, item):
+        """
+        Default value for attributes not explicetly created is None.
```

Comment:
```suggestion
        Default value for attributes not explicitly created is None.
```

Reply:
how many engineers does it take to write a single line of code which is at all a comment

Reply:
fun fact: if you drop this comment, you will need a line of code here :)